delete.php

```diff
@@ -0,0 +1,27 @@
+<?php
+    if (isset($_POST['noDelete'])) {
```

Comment:
if your linking off of the page anyway use elseif so all the if statements don't need to run.